tools/src/main/java/org/apache/kafka/tools/TopicCommand.java

```diff
@@ -799,6 +820,11 @@ public TopicCommandOptions(String[] args) {
                 "if set when creating topics, the action will only execute if the topic does not already exist.");
             excludeInternalTopicOpt = parser.accepts("exclude-internal",
                 "exclude internal topics when running list or describe command. The internal topics will be listed by default");
+            partitionSizeLimitPerResponseOpt = parser.accepts("partition-size-limit-per-response",
+                "the maximum partition size to be included in one DescribeTopicPartitions response. Only valid if use-describe-topics-api is used")
```

Comment:
Where is `use-describe-topics-api` defined?

Reply:
Good catch. removed.